src/System.Linq.Expressions/tests/Call/CallTests.cs

```diff
@@ -696,6 +696,37 @@ public static void EnumReturnType2()
         private static DayOfWeek ToDayOfWeek1(int i) => (DayOfWeek)i;
         private static DayOfWeek ToDayOfWeek2(int i, int j) => (DayOfWeek)(i + j);
 
+        [Fact]
+        public static void NullableEnumReturnType0()
+        {
+            Expression<Func<DayOfWeek?[]>> expr = () => new[] { ToDayOfWeekOpt0() };
+
+            Assert.Equal(DayOfWeek.Monday, expr.Compile(false)()[0].Value);
+            Assert.Equal(DayOfWeek.Monday, expr.Compile(true)()[0].Value);
```

Comment:
I just noticed this pattern in the previous tests you added (in addition to here).  In other places in this test suite, this has been accomplished with a theory, e.g.
https://github.com/dotnet/corefx/blob/9f6e8c72ea005abcd5680b7e5d325548874e899f/src/System.Linq.Expressions/tests/Call/CallTests.cs#L54-L56
Would it make sense to do the same thing for all of these?